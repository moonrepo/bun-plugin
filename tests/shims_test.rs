use proto_pdk_test_utils::{create_plugin, generate_global_shims_test};
use starbase_sandbox::{assert_snapshot, create_empty_sandbox};

generate_global_shims_test!("bun-test", ["bunx"]);
