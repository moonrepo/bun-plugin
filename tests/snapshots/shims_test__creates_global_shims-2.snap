---
source: tests/shims_test.rs
expression: "std::fs::read_to_string(sandbox.path().join(\".proto/bin\").join(\"bunx\")).unwrap()"
---
#!/usr/bin/env bash
set -e
[ -n "$PROTO_DEBUG" ] && set -x



exec proto run bun-test -- x "$@" 


